packages/models-library/src/models_library/projects_nodes_ui.py

```diff
@@ -0,0 +1,12 @@
+"""
+    Models node UI (legacy model, use instead projects.ui.py)
```

Comment:
Position CAN be used. The deprecation affect on WHERE is it used.
Before = directly each node
Now = there is a ui section

since we still keep the entry in nodes (marked as deprecated) and did not want to have cyclic imports. This was moved here.

when the deprecation note take finally effect, we will remove it from here